src/app/public/modules/repeater/repeater.component.ts

```diff
@@ -94,6 +107,13 @@ export class SkyRepeaterComponent implements AfterContentInit, OnChanges, OnDest
         }
       });
 
+    this.repeaterService.activeItemIndexChange
+      .takeUntil(this.ngUnsubscribe)
+      .subscribe((index: number) => {
+        this.activeIndex = index;
+        this.activeIndexChange.emit(index);
```

Comment:
Just to confirm: this will only emit when the previous `activeIndex` and the new `activeIndex` are different?

Reply:
Nope, it emits every time. Added an equality check and a unit test to check for this. Good catch.